wire-ios-request-strategy/Sources/Request Strategies/User Clients/FetchingClientRequestStrategy.swift

```diff
@@ -239,7 +239,8 @@ final class UserClientByUserClientIDTranscoder: IdentifierObjectSyncTranscoder {
         return ZMTransportRequest(path: path, method: .get, payload: nil, apiVersion: apiVersion.rawValue)
     }
 
-    public func didReceive(response: ZMTransportResponse, for identifiers: Set<UserClientID>) {
+    public func didReceive(response: ZMTransportResponse, for identifiers: Set<UserClientID>, completionHandler: @escaping () -> Void) {
+        defer { completionHandler() }
```

Comment:
Can't use defer. We should call this after the Task completes and in all other cases